schema.json

```
@@ -86,7 +86,7 @@
     "created": "2022-07-04",
     "updated": "2022-04-02",
     "homepage": "https://github.com/LaMednum/standard-mediation-num",
-    "uri": "https://raw.githubusercontent.com/LaMednum/standard-mediation-num/v0.1.0/schema.json",
+    "uri": "https://raw.githubusercontent.com/LaMednum/standard-mediation-num/v0.2.0/schema.json",
```

```suggestion
    "uri": "https://raw.githubusercontent.com/LaMednum/standard-mediation-num/v1.0.0/schema.json",
```
Logiquement si on fait des changements majeurs on augmente le premier chiffre et on remet les autres à 0. Là comme ce n'est plus compatible avec la version précédente je pense qu'on peut parler de montée de version majeur.
`0.2.0` c'est bien pour indiquer que l'on a ajouter un truc, mais que ça reste compatible (comme juste ajouter un nouveau champ).

Pour en savoir plus : https://semver.org/lang/fr/